database_cleanup/views/menu.xml

```diff
@@ -52,5 +52,18 @@
             <field name="parent_id" ref="menu_database_cleanup"/>
         </record>
 
+        <record model="ir.ui.menu" id="menu_create_indexes">
+            <field name="name">Create missing indexes</field>
+            <field name="sequence" eval="70" />
+            <field name="action" ref="cleanup_create_indexes_wizard_action" />
+        </record>
```

Comment:
Is this not missing the parent_id element? This will create top level menu entry...